Tuist/Templates/mvm/Project.stencil

```diff
@@ -0,0 +1,26 @@
+import ProjectDescription
+
+let project = Project.project(name: "{{ name }}")
+
+extension Project {
+
+    static func project(name: String) -> Project {
+        return Project(
+            name: name,
+            organizationName: "Nimble",
+            targets: [
+                Target(
+                    name: name,
+                    platform: .iOS,
+                    product: .app,
+                    bundleId: "co.nimblehq.\(name)",
+                    infoPlist: .default,
+                    sources: [
+                        "Sources/**"
+                    ]
+                )
+            ],
+            schemes: []
+        )
+    }
+}
```

Comment:
Can we separate these Swift codes out of this `.stencil` file, and use `.swift` file instead? How about `Project+Initializing.swift`?